pkgs/applications/audio/spotify/default.nix

```diff
@@ -8,7 +8,7 @@ let
   # Please update the stable branch!
   # Latest version number can be found at:
   # http://repository-origin.spotify.com/pool/non-free/s/spotify-client/
```

Comment:
Can you also add a warning to not use the beta versions found their. I think the least upgrade was accidentally.